weaveexec/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM gliderlabs/alpine
+MAINTAINER Weaveworks Inc <help@weave.works>
+WORKDIR /home/weave
+RUN ["apk", "add", "--update", "ethtool", "conntrack-tools", "curl", "iptables", "iproute2"]
+RUN ["sh", "-c", "rm -rf /var/cache/apk/*"]
+ADD ./weave /home/weave/
+ADD ./docker.tgz /
```

Comment:
wow, I didn't know docker was uncompressing .tgz in an 'ADD'. How gross. And `/` is on the path for the weave script execution, presumably?